pkg/cloudprovider/aws/suite_test.go

```diff
@@ -19,25 +19,27 @@ import (
 	"strings"
 	"testing"
 
-	"github.com/awslabs/karpenter/pkg/apis/provisioning/v1alpha1"
+	"github.com/aws/aws-sdk-go/aws"
+	"github.com/aws/aws-sdk-go/service/ec2"
+	. "github.com/awslabs/karpenter/pkg/test/expectations"
+
+	"github.com/Pallinder/go-randomdata"
```

Comment:
These imports look funny, and are imported differently than other files. 
IIUC our imports usually follow the following order:

```
go-native imports

karpenter imports

third party imports
```
My formatter just orders imports with no newline in between them. Maybe your does something differently.

Reply:
I think the dot import on expectations has a slightly different format. This is what my editor expects.